src/header-navigation/examples-list.js

```diff
@@ -0,0 +1,16 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+
+/* eslint-env node */
+/* eslint-disable flowtype/require-valid-file-annotation */
+module.exports = {
+  WITH_MENU_ELEMENT: 'With Dropdown Menu',
```

Comment:
can you please add `Header navigation` (or something similar) to the names, so for the visual regression tests it is simpler to include in `screener.config.js`?

Reply:
done